### DISCUSSION TITLE
Updating the roadmap

### DISCUSSION THREAD

**Discussion by Iey4iej3** [saber-notes/saber#675] [General]
It seems to me that, in the [roadmap](https://github.com/adil192/saber/discussions/1), *all* short term features and *most* of long term features are implemented. It seems to be time to update the roadmap accordingly?

**Comment by adil192** [comment]
Most of the "todo"s currently are filed as issues, and I pin the most important ones to the top. I think there's merit in providing an updated high-level roadmap for less technical users (probably as a section in the wiki), but I don't know when/if I'll get around to it.
